### PR TITLE
adding links

### PR DESCRIPTION
**Pull request by rbitia**
Adding a couple links to the Requirements section for the getting-started.md guide.

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -32,9 +32,9 @@
 - [kubectl]
 - [kustomize]
 - make
-- gettext (with `envsubst` in your PATH)
+- [gettext](https://www.gnu.org/software/gettext/) (with `envsubst` in your PATH)
 - md5sum
-- bazel
+- [bazel](https://docs.bazel.build/versions/master/install-os-x.html#install-on-mac-os-x-homebrew)
```
Comment (CecileRobertMichon) [inline docs/getting-started.md]: should we keep the link general for users who aren't on mac os (eg. Linux)?

Comment (rbitia) [reply docs/getting-started.md]: True, I can update!

Comment (CecileRobertMichon) [reply docs/getting-started.md]: /hold

File: docs/getting-started.md [hunk 2]
```diff
@@ -15,7 +15,7 @@
   - [Setting up the environment](#setting-up-the-environment)
   - [Generating cluster manifests and example cluster](#generating-cluster-manifests-and-example-cluster)
     - [Customizing the cluster deployment](#customizing-the-cluster-deployment)
-    - [Running the manifest generation script](#running-the-manifest-generation-script)
+    - [Running the manifest generation script](#running-the-manifest-generation-scr ipt)
```
Comment (CecileRobertMichon) [inline docs/getting-started.md]:
```suggestion
    - [Running the manifest generation script](#running-the-manifest-generation-script)
```

**Comment by CecileRobertMichon** [comment]
/hold cancel

@rbitia you need to fix https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/343#issuecomment-566161316 for the PR to be able to merge

**Pull request merged**